L/LinearElasticity/build_tarballs.jl

```diff
@@ -0,0 +1,74 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder
+
+name = "LinearElasticity"
+version = v"5.0"
+
+# Collection of sources required to build MMG
+sources = [
+    GitSource("https://github.com/ISCDtoolbox/LinearElasticity.git",
+              "58ace131fd1af9293ded9f84f2a6447147a8ecdc"), # master @ May 6, 2021
+    GitSource("https://github.com/ISCDtoolbox/Commons.git",
+              "57f0b0ed46d45a12fa60c42bce2c6fa73c84bc30"), # master @ Jul 23, 2021
+    DirectorySource("./bundled")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd ${WORKSPACE}/srcdir/Commons
+if [[ "${target}" == *mingw* ]]; then
+    atomic_patch -p1 "${WORKSPACE}/srcdir/patches/Commons.mingw.patch"
+elif [[ "${target}" == *freebsd* ]]; then
+    atomic_patch -p1 "${WORKSPACE}/srcdir/patches/Commons.freebsd.patch"
+else
+    atomic_patch -p1 "${WORKSPACE}/srcdir/patches/Commons.patch"
+fi
+mkdir build
+cd build
+cmake .. \
+    -DCMAKE_INSTALL_PREFIX=${prefix} \
+    -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+    -DCMAKE_BUILD_TYPE=Release \
+    -DBUILD_SHARED_LIBS=ON
+make -j${nproc}
+make install
+
+cd ${WORKSPACE}/srcdir/LinearElasticity
+if [[ "${target}" == *mingw* ]]; then
+    atomic_patch -p1 "${WORKSPACE}/srcdir/patches/LinearElasticity.mingw.patch"
+else
+    atomic_patch -p1 "${WORKSPACE}/srcdir/patches/LinearElasticity.patch"
+fi
+if [[ "${target}" == *freebsd* ]]; then
+    LDFLAGS="-lpmc -lc++"
+fi
```

Comment:
Why?

Reply:
In the Commons dependency for FreeBSD builds, there are some functions from the `pmc` library used which lead to linker errors for undefined references when building LinearElasticity without those extra link flags. I agree it isn't very nice, any suggestions on a better way to resolve?

Reply:
This is more nicely resolved by adding the linker dependencies directly to Commons in the CMakeLists.txt file. Resolved in 612be31.